Content.Server/Chemistry/ReagentEffects/MakeSentient.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+//using Content.Server.AI.Components; // This was in the original code but caused an error so I'm gonna keep it as a comment just in case
+using Content.Server.Mind.Components;
+using Content.Server.NPC.Components; // Don't think I actually need this but keeping it just in case
+using Content.Shared.Emoting;
+using Content.Shared.Examine;
+using Content.Shared.Movement.Components;
+using Content.Shared.Speech;
+using Content.Shared.Damage;
+
+using Content.Shared.Chemistry.Reagent;
+using Content.Server.Ghost.Roles.Components;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed class MakeSentient : ReagentEffect
+{
+    public override void Effect(ReagentEffectArgs args)
+    {
+        var entityManager = args.EntityManager;
+        var uid = args.SolutionEntity;
+
+        // This makes it so it doesn't affect things that are already sentient
+        if (entityManager.HasComponent<MindComponent>(uid))
+        {
+            return;
+        }
+
+        // This removes the ActiveNPCComponent, because I'm pretty sure that gets rid of the preexisting AI which we probably don't want
+        // Any glitches found in this code probably stem from this part
+        entityManager.RemoveComponent<ActiveNPCComponent>(uid);
```

Comment:
This will be redundant as of https://github.com/space-wizards/space-station-14/pull/12014

Reply:
Alright, I removed it. Hopefully these both get merged at the same time, or there might be a couple glitches happening. Either way, it should be fine.